index.bs

```diff
@@ -142,14 +142,15 @@ object store and indexes. Finally, the opened connection is saved for
 use in subsequent examples.
 
 ```js
-var request = indexedDB.open("library");
+const request = indexedDB.open("library");
+let db;
 
 request.onupgradeneeded = function() {
   // The database did not previously exist, so create object stores and indexes.
-  var db = request.result;
-  var store = db.createObjectStore("books", {keyPath: "isbn"});
-  var titleIndex = store.createIndex("by_title", "title", {unique: true});
-  var authorIndex = store.createIndex("by_author", "author");
+  const db = request.result;
```

Comment:
Should const be here? I believe the goal was to set the variable in the outer scope context.

Reply:
Oh, crud, that should have been in another branch. I screwed up my branch history. Let me minimize the diff.